src/components/Autosuggest/Autosuggest.js

```diff
@@ -328,8 +329,9 @@ export class Autosuggest extends React.Component {
 
         const stateAndProps = { ...this.props, ...this.state };
         const hideInputPlaceholder = !focused && !!selectedPlaceholder;
-        const doShowClearButton =
-            showClearButton && !!selectedSuggestions && !!selectedSuggestions.length && !focused;
+        const doShowClearButton = saveSelectedValueToInput
+            ? showClearButton && inputValue.length > 0 && !focused
```

Comment:
When the field is focused, Clear button is not visible.
Maybe I'm missing something, and there was a discussion behind it, but for me it feels like a bug.
Why a user can't clear the field when it is focused?

Reply:
Unfortunately it is not a straight-forward solution and it requires some breaking changes or updates in structuring of Autosuggest component. I noted that down to [REACH-278](https://jira.textkernel.nl/browse/REACH-278) so we can continue this discussion from there.